hedera-node/hedera-mono-service/src/main/java/com/hedera/services/contracts/ContractsModule.java

```diff
@@ -161,8 +163,11 @@ static ContractValidationRule providePrefixCodeRule() {
     static MessageCallProcessor provideV_0_30MessageCallProcessor(
             final @V_0_30 EVM evm,
             final @V_0_30 PrecompileContractRegistry precompiles,
-            final Map<String, PrecompiledContract> hederaPrecompileList) {
-        return new HederaMessageCallProcessor(evm, precompiles, hederaPrecompileList);
+            final Map<String, PrecompiledContract> hederaPrecompileList,
```

Comment:
I consider supporting Lazy Creation to be an EVM alteration.  So v0.30 should not support it, we will need flags in the operations or new subtypes.

Reply:
Added an EVM version check now in the call transition logic, so if version == 0.30 we fail as we have always failed up to this point and not even reach the EVM. I think we can manage with only this check for now, though I’m not 100% sure about the approach taken for the `HederaMessageCallProcessor`?

When we add support for lazy creates through .`call()` I think we will need to subclass/add flags in `HederaCallOperation`.